sdks/python/apache_beam/runners/worker/operations.py

```diff
@@ -296,10 +297,11 @@ class DoOperation(Operation):
 
   def __init__(
       self, name, spec, counter_factory, sampler, side_input_maps=None,
-      user_state_context=None):
+      user_state_context=None, timer_inputs=None):
     super(DoOperation, self).__init__(name, spec, counter_factory, sampler)
     self.side_input_maps = side_input_maps
     self.user_state_context = user_state_context
+    self.timer_inputs = timer_inputs or {}
```

Comment:
Can you comment in the pydoc what this is supposed to be, especially since this is never really used in this file?